components/cluster/command/deploy.go

```diff
@@ -353,14 +354,14 @@ func deploy(clusterName, clusterVersion, topoFile string, opt deployOptions) err
 func buildMonitoredDeployTask(
 	clusterName string,
 	uniqueHosts map[string]hostInfo, // host -> ssh-port, os, arch
-	globalOptions meta.GlobalOptions,
-	monitoredOptions meta.MonitoredOptions,
+	globalOptions spec.GlobalOptions,
+	monitoredOptions spec.MonitoredOptions,
```

Comment:
If dms use the similar GlobalOptions, should dms rewrite another one?

Reply:
I think eventually `GlobalOptions` will be different with DMS (MonitoredOpions/Grafana/Prometheus/AlertManager will be the same, but we can leave them to the future PR).